### DISCUSSION TITLE
I don't see anybody assignee to review for my team from codeowner file

### DISCUSSION THREAD

**Discussion by splincode** [community/community#141048] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have valid codeowner file

<img width="418" alt="image" src="https://github.com/user-attachments/assets/398d848e-aab7-48d0-bf35-b5553960e362">

https://github.com/taiga-family/renovate-config/blob/main/.github/CODEOWNERS

But after PR, I don't see anybody assignee to review, why?

https://github.com/taiga-family/renovate-config/pull/1478

<img width="391" alt="image" src="https://github.com/user-attachments/assets/af432505-cd3f-4b25-8bb0-d41fc106ad68">

My setting for team

<img width="525" alt="image" src="https://github.com/user-attachments/assets/0e5abb52-3526-4796-8409-6b4c72986a8f">
<img width="726" alt="image" src="https://github.com/user-attachments/assets/62c93ed9-6fcf-4d6a-be5e-07dc2a08c079">

**Comment by BDadmehr0** [comment]
Suggested Actions:

1. Double-check your `CODEOWNERS` file [here](https://github.com/taiga-family/renovate-config/blob/main/.github/CODEOWNERS) to ensure everything is properly formatted and valid.
2. Make sure the team has the appropriate repository permissions.
3. Ensure the PR is not a draft and is targeting a branch covered by the `CODEOWNERS` rules.

**Reply by splincode** [reply]
Hello

1. Double-check

<img width="428" alt="image" src="https://github.com/user-attachments/assets/bf71afd8-19eb-4f01-ab07-dea35ecb6c47">


2. Yes, all maintainers are admins

<img width="639" alt="image" src="https://github.com/user-attachments/assets/7f5147e7-3444-4eea-978f-285d7d5e9d11">
<img width="815" alt="image" src="https://github.com/user-attachments/assets/09c82ec8-d596-484c-aa06-1f3970c652a4">

3. My Pull request aren't draft

https://github.com/taiga-family/editor/pull/1531

**Reply by MG72021** [reply]
[google.com](url)

**Comment by BDadmehr0** [comment]
Check if the PR is targeting a branch that is actually listed in the CODEOWNERS file. Sometimes, if a branch isn’t covered by the file, the assignments won’t happen.
Ensure the team is explicitly listed in the CODEOWNERS file. Sometimes misconfiguration can lead to teams not being recognized.
Check organization permissions: Even if maintainers are admins, there could be higher-level restrictions on the organization or repo that need adjusting.

**Reply by splincode** [reply]
1. `Check if the PR is targeting a branch that is actually listed in the CODEOWNERS file`

Checked, everything fine with it

2. `if a branch isn’t covered by the file`

My `codeowner` file has single line for `any`

```
* @taiga-family/core-team
```

3. `Check organization permissions`

My team is All-repository admin for organization
![image](https://github.com/user-attachments/assets/90ac6a8f-9aff-4dfc-b040-23a01a3c3f2f)

And in my repo settings I see that my team have permissions

![image](https://github.com/user-attachments/assets/a1a6dd35-6876-435a-a929-39416aa1d2cd)

**Comment by BDadmehr0** [comment]
You’ve already confirmed that everything is set up correctly with the `CODEOWNERS` file and your team has admin permissions for the repository. So, the issue might be something else. Here are a few more things to check:

### 1. **Check Branch Protection Rules**:
   - Make sure there are no branch protection rules that might interfere with the `CODEOWNERS` file. For example, sometimes specific reviewer requirements can override the automatic assignments from `CODEOWNERS`.

### 2. **Double-check the Team Name**:
   - Verify that the team name in the `CODEOWNERS` file (`@taiga-family/core-team`) is correct. Even small differences in spacing, capitalization, or spelling can cause issues.

### 3. **Repository Visibility**:
   - If your repository is private, confirm that `CODEOWNERS` is working for private repos in your organization. Sometimes private repos have special limitations or require additional settings for auto-assigning reviewers.

### 4. **Check GitHub Actions/Automation**:
   - If you are using bots or GitHub Actions to manage PRs, make sure they are not interfering with the `CODEOWNERS` functionality. Automated workflows can sometimes alter PR behavior in a way that prevents assignments.

### 5. **PR-Specific Settings**:
   - Ensure there are no manual configurations in the specific pull request (PR) that are disabling automatic reviewer assignments. For example, a PR might have been manually set up to ignore `CODEOWNERS` rules.

### 6. **GitHub Caching**:
   - Sometimes GitHub caches settings. Try making a small edit to the `CODEOWNERS` file, commit it, and then reopen or create a new PR to see if this forces GitHub to reapply the settings correctly.

If none of these steps solve the issue, it might be necessary to reach out to GitHub support, as it could be a deeper configuration or platform issue.

**Reply by splincode** [reply]
When I open pull request I see
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/27d1fcf5-2694-4542-ab6d-669117a17af3">

But after finished all pipelines I don't see any reviewers 

https://github.com/taiga-family/editor/pull/1532

**Reply by BDadmehr0** [reply]
Based on the screenshot, it looks like when you open the pull request (PR), the @taiga-family/core-team is correctly shown as the reviewer. However, after the pipelines are finished, you mention that no reviewers are actually assigned.

Here are a few more things to check:
1. Check if the PR is in Draft Mode:

    Sometimes when a PR is still in draft mode, reviewers are not automatically assigned until it is marked as "Ready for Review." Make sure the PR is not in draft mode.

2. Ensure the Pipeline Doesn't Interfere with Review Assignment:

    It’s possible that something in the pipeline (CI/CD process) could be removing or resetting the assigned reviewers. Check if any steps in the pipeline or automation are altering the PR after it is created.

3. Verify Required Reviewers in Branch Protection Rules:

    If branch protection rules are in place, make sure that @taiga-family/core-team is set as a required reviewer. If this is not set, it could be that the automatic assignment is getting removed or overlooked.

4. Manual Assignment:

    After the pipelines finish, are you able to manually assign the reviewers? If so, it might indicate an issue with GitHub’s automatic assignment or a conflict in the repository settings.

5. Check for GitHub Webhooks or Apps:

    If you are using any GitHub Apps, webhooks, or bots, check whether they might be overriding the automatic reviewer assignments after the PR is created.

**Reply by BDadmehr0** [reply]
If none of these solve the issue, it could be useful to reach out to GitHub support with the exact behavior you're seeing after the pipelines complete, as this may be an unusual platform-specific problem.

**Reply by splincode** [reply]
`Verify Required Reviewers in Branch Protection Rules`

I turned these settings on and off and still no effect

<img width="759" alt="image" src="https://github.com/user-attachments/assets/fdc9efa6-1b61-4186-94b3-896a98e98eeb">

`Check for GitHub Webhooks or Apps`

i have standard github app for frontends, I don't have idea what's wrong with that can be

<img width="822" alt="image" src="https://github.com/user-attachments/assets/d5475743-f6cd-4290-92f2-5946de1579e6">


`it could be useful to reach out to GitHub support `

Oh, thank you, I open ticket on github support board

**Reply by BDadmehr0** [reply, marked as answer]
Please, I hope this issue gets resolved, and if my response was helpful, please confirm it.

**Reply by rahmat-archive1** [reply]
@splincode Hi, may I know if there is any update on your github support board ticket? We faced similar issue and looking for the solution. Hopefully this has been resolved and appreciate if you can share the result

**Reply by splincode** [reply]
Yes, it turns out that you still need to manually add each repository to the team, it's annoying, but working

<img width="763" alt="image" src="https://github.com/user-attachments/assets/3db6da84-2b3e-4d01-a6e6-c998291494d4">
